src/system.h

```diff
@@ -89,6 +89,7 @@ struct System {
     bool fswatch;
     bool autohtmlexport;
     bool casesensitivesearch;
+    bool _darkennonmatchingcells;

```

Comment:
why does this have an `_` ?

Reply:
This was indeed a remnant from the previous proposal. Corrected it.